Reference/Templating/Mvc/views.md

```diff
@@ -10,82 +10,56 @@ _Working with MVC Views and Razor syntax in Umbraco_
 
 ## Properties available in Views
 
-All Umbraco views inherit from `Umbraco.Web.Mvc.UmbracoTemplatePage` which exposes many properties that are available in razor:
+All Umbraco views inherit from `Umbraco.Web.Mvc.UmbracoViewPage<ContentModels.NameOfYourDocType>` along with the using statement `@using ContentModels = Umbraco.Web.PublishedModels;`. This exposes many properties that are available in razor,. This means that properties on the Document Type can be accessed in a number of ways:
 
+* @Model (of type `Umbraco.Web.Mvc.RenderModel`) -> the model for the view which contains the standard list of IPublishedContent properties but also gives you access to the typed current page (of type whatever type you have added in the angled brackets).
 * @Umbraco (of type `Umbraco.Web.UmbracoHelper`) -> contains many helpful methods, from rendering macros and fields to retrieving content based on an Id and tons of other helpful methods. [See UmbracoHelper Documentation](../../Querying/UmbracoHelper/index.md)
 * @Html (of type `HtmlHelper`) -> the same HtmlHelper you know and love from Microsoft but we've added a bunch of handy extension methods like @Html.BeginUmbracoForm
-* @CurrentPage (of type `DynamicPublishedContent`) -> the dynamic representation of the current page model which allows dynamic access to fields and also dynamic LINQ
-* @Model (of type `Umbraco.Web.Mvc.RenderModel`) -> the model for the view which contains a property called `Content` which gives you access to the typed current page (of type `IPublishedContent`).
 * @UmbracoContext (of type `Umbraco.Web.UmbracoContext`)
-* @ApplicationContext (of type `Umbraco.Core.ApplicationContext`)
 * @Members (of type `Umbraco.Web.Security.MemberShipHelper`) [See MemberShipHelper Documentation](../../Querying/MemberShipHelper/index.md)
 
-## Rendering a field with UmbracoHelper
+## Rendering a field in a strongly typed view
 
-This is probably the most used method which renders the contents of a field for the current content item.
+This is probably the most used method which renders the contents of a field for the using the alias of the content item.
 
 ```csharp
-@Umbraco.Field("bodyContent")
+@Model.Value("bodyContent")
 ```
 
-If you're using the Field method from within a partial view then be aware that you will need to pass the context so the Field method knows where to get the desired value from. For instance you can pass "CurrentPage" like this:
+If you're using the method from within a partial view then be aware that you will need to inherit the context so the method knows which type to get the desired value from. You'd do this at the top of partial view and so strongly typed properties can then be accessed in the partial view. For instance you can pass "HomePage" like this:
 
 ```csharp
-@Umbraco.Field(Model.Content, "eventLink")
+@inherits UmbracoViewPage<SiteSettings>
+...
+@Model.Value("title")
```

Comment:
If we're showing people the wiring up of the view using the Modelsbuilder approach... eg
@inherits UmbracoViewPage<BlogPost>

should we show the first rendering out of a field (and are we still using the term field to refer to properties?) using a more Modelsbuildery technique eg

@Model.BodyContent

instead of 

@Model.Value("bodyContent")

Don't get me wrong I much prefer the simpler non-modelsbuilder approach... but as it's 'on' by default - would it make sense to people new to Umbraco to show that methodology first...

... but also showing the Model.Value("") way too?

It depends a little on how further the codebase for Modelsbuilder becomes part of the core and not a plugin

There is a section on Modelsbuilder at the end of the article
but that's probably a bigger question than just updating the info to see if it works with V8 :-)

Reply:
I agree. I felt as though I'd keep the structure the same so begin with Model.Value although it's something I wouldn't really use... I'm not sure int he value (no pun intended) of doing things this way. I'm going to get Modelsbuilder at the forefront - it's my preferred way but now I was reluctant at first to do it. As it's out of the box now, I think there's no danger in doing it. It's actually simpler than the 'simple' way.

Reply:
The syntax for writing out a property is simpler with Modelsbuilder... however the concept of generated models, and extending them along with the inherits statement being different at the top of each view can be a little more confusing when you are getting started... (depending on what you already know of course in terms of C# and MVC!) - you don't get caught out with the Models being out of sync between environments if you use: Model.Value("bodyContent")... (stringly typed models :-)) and if you use UmbracoViewPage everywhere, then you don't run into problems with the wrong model being passed to a shared partial view etc... also the 'Insert' menu  and Query builder use the @Model.Value approach 
![image](https://user-images.githubusercontent.com/260802/67622754-347b2c80-f815-11e9-8591-bda276266e6d.png) and generate 'queries' without the Modelsbuilder syntax - so if you utilise these which is more common when starting out, it's weird if this isn't mentioned in the documentation!

The conclusion we came to in docs discussions was that both methods should probably be presented equally, but perhaps we always mention 'Modelsbuilder' first since that is the way we think HQ wants people to go... (eg pure live being on by default, and the work to put modelsbuilder 'lite' into the core) but the other approach is 'simpler' conceptually... (it depends I think on the context of the project - Umbraco has such a broad spectrum of usage there isn't one best way! - but at least now we are down to two good ways instead of 5 ways!)

Reply:
Fab - that clears it all up a little for me. I agree that 2 ways is better than 5 too!